server/test/stubs/users.stub.ts

```diff
@@ -0,0 +1,25 @@
+import { CreateUserDto } from '../../src/users/dto/create-user.dto';
+import { User } from '../../src/users/entities/user.entity';
+import { assetsStub } from './assets.stub';
+import { messageStub } from './messages.stub';
+
+export const userEntityStub = (): User => {
+  return {
+    id: 234545,
+    first_name: 'fakeUser',
+    last_name: 'John',
+    email: 'Doe',
+    password: 'Secret1234$',
+    assets: [assetsStub()],
+    messages: [messageStub()],
+  };
+};
```

Comment:
maybe something like this to prevent circular deps?
```
export const userEntityStub = (
  asset?: Asset = assetsStub(),
  message?: Message = messageStub(),
): User => {
  return {
...
    assets: [asset],
    messages: [message],
  };
};
```